eq-publisher/src/eq_schema/builders/routing2/translateBinaryExpression.js

```diff
@@ -25,11 +29,16 @@ const buildRadioAnswerBinaryExpression = ({ left, right }) => {
 };
 
 const buildCheckboxAnswerBinaryExpression = ({ left, right, condition }) => {
-  return {
+  let returnVal = {
```

Comment:
I think this can be const

Reply:
Ooh. I didn't realise that worked. Cheers!